README.md

```diff
@@ -152,6 +152,15 @@ To run the above benchmarks on your machine, see instructions in the [examples](
 
 ## Installation
 
+### Conda Packages
+
+We provide conda packages for [the most recent release](https://anaconda.org/aihabitat) and built [nightly for the master branch](https://anaconda.org/aihabitat-nightly). Packages are provided for standard and headless systems with and without Bullet physics support. When possible, we suggest this method of installation for the most simple and straight-forward installation experience. To learn more about building your own conda packages for Habitat-sim, checkout the [conda-build README](https://github.com/facebookresearch/habitat-sim/tree/master/conda-build).
```

Comment:
```suggestion
We provide conda packages for [the most recent release](https://anaconda.org/aihabitat) and built [nightly for the master branch](https://anaconda.org/aihabitat-nightly). Packages are provided for standard and headless systems with and without Bullet physics support. When possible, we suggest this method of installation for the most simple and straight-forward installation experience.
```

That readme is just for building habitat sim's conda package. 99% of users don't need to read it.